pkg/R/RDD.R

```diff
@@ -21,9 +24,16 @@ setClass("PipelinedRDD",
                       prev_jrdd = "jobj"),
          contains = "RDD")
 
-
-setMethod("initialize", "RDD", function(.Object, jrdd, serialized,
-                                        isCached, isCheckpointed) {
+setMethod("initialize", "RDD", function(.Object, jrdd, serializedMode,
+                                        isCached, isCheckpointed, colNames) {
+  # Check that RDD constructor is using the correct version of serializedMode
+  stopifnot(class(serializedMode) == "character")
+  stopifnot(serializedMode %in% c("byte", "string", "row"))
```

Comment:
I prefer `byte` in this case solely because it makes the code a lot clear at a glance. `row` and `raw` look very similar when you're skimming.